src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,24 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import React from 'react';
+import classNames from 'classnames';
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const { completed } = todo;
+
+  return (
+    <article className={
+      classNames('TodoInfo', { 'TodoInfo--completed': completed })
+    }
+    >
+      <h2 className="TodoInfo__title">
+        {todo.title}
```

Comment:
you've already started to destructure todo object. So make the same for other keys
```suggestion
        {title}
```